.github/workflows/build.yml

```diff
@@ -0,0 +1,37 @@
+name: Runs Preetier Check, Lint Check, Build Check
+# first runs prettier and lint checks then builds the code.
+on: [pull_request, push]
+
+jobs:
+  build-project:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+
+      - name: Install Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: 18
+
+      - name: Install Dependencies
+        run: npm install
+
+      - name: Code Formatting Check #Runs Prettier Script
+        run: npm run prettier
+
+      - name: Linting Check #Runs ESLint Script
+        run: npm run lint
+
+      - name: Build #Build the project
+        run: npm run build
+
+      - name: Check Build Status
+        run: |
+          if [ $? -eq 0 ]; then
+            echo "Build successful!"
+          else
+            echo "Build failed!"
+            exit 1
+          fi
```

Comment:
Let's try to combine these two `.yml` files into one. It'll allow us to define an order in which both of them will execute, and we'll use the same code to trigger the script.

To do that all you need to do is to migrate the code from line number 9 to 30 of `chromatic.yml` to this file. Make sure the indentation is such that:

```yml
jobs:
   build-project:
      name: Run Prettier, Lint and Build Check
      // your first amazing automation script
   chromatic-deployment:
      name: Deploy StoryBook on Chromatic
      // your second amazing automation script
```

You can use the `name:` attribute to give name to the two jobs.  Once you do that, please rename this file to `actions.yml`.

You can use the `git mv` command to do that.

Reply:
ok sure!